skills/declarative/calendarSkill/calendarskill.dialog

```diff
@@ -1280,20 +1280,1154 @@
         }
       ]
     },
+    {
```

Comment:
It looks likes some of these dialogs got duplicated

Reply:
Oops, it really a troublesome work to merging different breanch of adaptive dialog. Fixed